pkg/services/user/userimpl/store.go

```diff
@@ -64,6 +63,9 @@ func (ss *sqlStore) Insert(ctx context.Context, cmd *user.User) (int64, error) {
 			cmd.UID = util.GenerateShortUID()
 		}
 
+		cmd.Email = strings.ToLower(cmd.Email)
+		cmd.Login = strings.ToLower(cmd.Login)
```

Comment:
This was done so that we can be sure that stored email and login is always lower-case